products/arangodb

```diff
@@ -0,0 +1,33 @@
+---
+title: ArangoDB
+category: db
+permalink: /arangodb
+releasePolicyLink: https://arangodb.com/master-services-agreement-inc-may-2023-cs/#eolpolicy
+changelogTemplate: "https://docs.arangodb.com/stable/release-notes/version-__RELEASE_CYCLE__/"
+releaseLabel: "ArangoDB __RELEASE_CYCLE__"
+eolWarnThreshold: 121
+releases:
+-   releaseCycle: "3.10"
+    latest: "3.10.11"
+    releaseDate: 2022-10-04
+    eol: 2024-04-15
+-   releaseCycle: "3.11"
+    latest: "3.11.6"
+    releaseDate: 2023-05-30
+    eol: 2024-11-30
+-   releaseCycle: "3.12"
+    latest: "unreleased"
```

Comment:
We never document unreleased releases on endoflife.date, so 3.12 entry must be dropped.